lib/refinery_api.py

```diff
@@ -98,7 +98,8 @@ def build_timestamp(time, day):
         return None
     # extract 12 from '12:00'
     hour = int(time.split(':')[0])
-    return day.replace(hour=hour).isoformat()
+    minute = int(time.split(':')[1])
+    return day.replace(hour=hour, minute=minute).isoformat()
```

Comment:
Should we not also `replace` seconds and ms? The returned time should be zeros all the way down so that it represents the true canonical start and end times - not an approximation

Reply:
good point!